pkg/util/errors.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func IsNotFoundErr(err error) bool {
-	return k8serrors.IsNotFound(errors.Cause(err))
+	return k8serrors.IsNotFound(err)
```

Comment:
why remove this?

Reply:
Because it's unnecessary, and has the possibility to break things
The k8serrors code repeatedly calls errors.cause to find if any error matches

So all this code really does is ensure that if we pass it a raw isnotfounderror, we'll unwrap it and get something that doesn't match

Judging from the git blame, this wasn't intentional, and came from before the use of k8serrors.